### DISCUSSION TITLE
How do I add margin top and bottom to swiper-pagination div?

### DISCUSSION THREAD

**Discussion by RachelGnanaraj** [nolimits4web/swiper#4487] [Q&A]
I am unable to find a way to add margins above and below the swiper pagination div container. 
I tried adding this:
.swiper-pagination {
        margin-top: 24px !important;
        margin-bottom: 24px !important;
      }

but still this doesn't work. 
Any idea how do i achieve this?

A simple demo: [https://codesandbox.io/s/swiper-pagination-dynamic-forked-00v1i?file=/index.html:1304-1409]

**Comment by vltansky** [comment]
You should use `bottom` instead of `margin`:
![image](https://user-images.githubusercontent.com/5851280/116222348-32ba5e00-a757-11eb-8601-27abf55333c8.png)
